lib/shopify-cli/commands/connect.rb

```diff
@@ -4,16 +4,30 @@ module ShopifyCli
   module Commands
     class Connect < ShopifyCli::Command
       def call(*)
-        if Project.current
-          @ctx.puts @ctx.message('core.connect.production_warning')
-          org = fetch_org
-          id = org['id']
-          app = get_app(org['apps'])
-          shop = get_shop(org['stores'], id)
-          write_env(app, shop)
-          @ctx.puts(@ctx.message('core.connect.connected', app.first['title']))
-          @ctx.puts(@ctx.message('core.connect.serve', ShopifyCli::TOOL_NAME))
+        project_type = ask_project_type unless Project.has_current?
+
+        @ctx.puts @ctx.message('core.connect.production_warning') if Project.has_current? && Project.current
+
+        env_file = Resources::EnvFile.parse_external_env
```

Comment:
`env_file` suggests a `File` object... maybe this could be `env_hash` or `env_data`?

Reply:
Good point!